birdhouse/optional-components/weaver/default.env

```diff
@@ -0,0 +1,47 @@
+# ------
+# weaver
+# ------
+
+EXTRA_VARS='
+  $WEAVER_CONFIG
+  $WEAVER_MANAGER_NAME
+  $WEAVER_WORKER_NAME
+  $WEAVER_WPS_NAME
+  $WEAVER_WPS_OUTPUTS_DIR
+  $WEAVER_WPS_OUTPUTS_PATH
+  $WEAVER_WPS_PATH
+  $WEAVER_WPS_WORKDIR
+'
+
+# extend the original 'VARS' from 'birdhouse/pavics-compose.sh' to employ them for template substitution
+# adding them to 'VARS', they will also be validated in case of override of 'default.env' using 'env.local'
+VARS="$VARS $EXTRA_VARS"
+
+# configuration can be EMS|ADES
+# the corresponding "/<config>" route will redirect to the protected 'weaver' service
+export WEAVER_CONFIG=ADES
+# endpoint where Weaver will be accessible from
+export WEAVER_VERSION=1.14.0
+# real names of the weaver/worker services
+# these values will generate "<server-fqdn>/<name>" URI to access them behind secured proxy
+export WEAVER_MANAGER_NAME=weaver
+export WEAVER_WORKER_NAME=worker
+export WEAVER_WPS_NAME=${WEAVER_MANAGER_NAME}-wps
+# WPS-1/2 path under Weaver's root endpoint
+export WEAVER_WPS_PATH=/wps
+# data persistence of wps outputs (directory path)
+# for the moment, this is hardcoded in 'birdhouse/docker-compose.yml'
+export WEAVER_WPS_OUTPUTS_DIR="${DATA_PERSIST_ROOT}/wps_outputs/weaver"
```

Comment:
I'm not sure this would do the trick.
First, just to be clear `WEAVER_WPS_OUTPUTS_DIR` refers to a path used internally in weaver, while `DATA_PERSIST_ROOT` refers to a path of the host. Nothing prevent weaver from using the same root for data but I think it can cause useless confusion.

Ok, main course now ;) 
wps_outputs compose volume is mapped on the host like this : `wps_outputs:/pavics-data/wps_outputs`
Weaver map the wps_outputs volume like this : `wps_outputs:${WEAVER_WPS_OUTPUTS_DIR}`
So the weaver directory `${DATA_PERSIST_ROOT}/wps_outputs/weaver` will be mapped to the host directory `/pavics-data/wps_outputs` and weaver will write outputs directly under wps_outputs on the host.

We want weaver to write in the host directory `/pavics-data/wps_outputs/weaver`.
So we should have something like that : 

1. `export WEAVER_MAPPED_WPS_OUTPUTS_DIR="/data/wpsoutputs"`
2. Weaver mapping  `wps_outputs:${WEAVER_MAPPED_WPS_OUTPUTS_DIR}`
3. export WEAVER_WPS_OUTPUTS_DIR="${WEAVER_MAPPED_WPS_OUTPUTS_DIR}/weaver"

Reply:
I find there is a design problem in the stack itself if all data directories are accessible from any bird. 
Really, there should only be a mounting of `"/pavics-data/wps_outputs/weaver:/data/wps_outputs"` and weaver shouldn't worry about creating a sub-directory named `weaver` for itself. It should only dump in the specified location. 
I don't know of any way to combine named-volume with extra path. This can be done only with string mapping and variables.

Reply:
I think that volumes should be ok now. Please see notes I added in 7aa56f0. This is an extra difficulty that must be considered in the case of Weaver on top of the `/data/wpsoutputs/<bird>` format.